Bottom: de1d0b2128e490f4cc93ea30b4945fcfaff8f639
Top:    96b3631be4c66909ae6edf7518ef83b9abd4d916
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-22 20:09:24 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
